docs/dev/running/aziot-certd.md

```diff
@@ -211,7 +211,7 @@ With this basic file, fill it out depending on what workflow you want to test:
 
     export AZIOT_CERTD_CONFIG='...'
 
-    (exec -a aziot-certd target/x86_64-unknown-linux-gnu/debug/aziotd)
+    cargo run --target x86_64-unknown-linux-gnu -p aziotd -- aziot-certd
```

Comment:
Is there any reason you're explicitly stating the `--target` here? 
Is it just to make make extra sure that the `LD_LIBRARY_PATH` is correct?

Reply:
The docs tell you to build using `make` which always sets `--target` (for ease when cross-compiling), so if `--target` wasn't specified here it'd do the whole build from scratch.